drivers/gpio/Kconfig.xec

```diff
@@ -0,0 +1,54 @@
+# Kconfig.xec - Microchip XEC GPIO configuration options
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig GPIO_XEC
+	bool "XEC Microchip GPIO driver"
+	depends on GPIO
```

Comment:
Let's standardize with depends on SOC_FAMILY_MEC as agreed for i2C driver

Reply:
this was also fixed